corehq/apps/reports/static/reports/javascripts/config.dataTables.bootstrap.js

```diff
@@ -199,6 +199,13 @@ function HQReportDataTables(options) {
                 $('.dataTables_paginate a').on('click', function () {
                     datatable.fnAdjustColumnSizing();
                 });
+                // This fixes a bug in some browsers where if the first column
+                // contains a large amount of data, it will overlap with the
+                // second column. This makes sure after load, the columns are
+                // re-adjusted.
+                setTimeout( function () {
+                    datatable.fnAdjustColumnSizing();
```

Comment:
@biyeun do you happen to remember what browsers this affected? I tried both ff and chrome with this code commented out and didn't have any issues. (I don't think the bug in the commit message is related)